src/api/liskApi.js

```diff
@@ -258,191 +243,120 @@ LiskAPI.prototype.getAddressFromSecret = function getAddressFromSecret(secret) {
 /**
  * @method getAccount
  * @param address
- * @param optionsOrCallback
- * @param callbackIfOptions
  *
  * @return API object
  */
 
-LiskAPI.prototype.getAccount = function getAccount(
-	address, optionsOrCallback, callbackIfOptions,
-) {
-	const callback = callbackIfOptions || optionsOrCallback;
-	const data = this.constructRequestData({ address }, optionsOrCallback);
-	return this.sendRequest(GET, 'accounts', data, callback);
-};
+LiskAPI.prototype.getAccount = privateApi.wrapSendRequest(GET, 'accounts', address => ({ address }));
 
 /**
  * @method listActiveDelegates
  * @param limit
- * @param optionsOrCallback
- * @param callbackIfOptions
  *
  * @return API object
  */
 
-LiskAPI.prototype.listActiveDelegates = function listActiveDelegates(
-	limit, optionsOrCallback, callbackIfOptions,
-) {
-	const callback = callbackIfOptions || optionsOrCallback;
-	const data = this.constructRequestData({ limit }, optionsOrCallback);
-	return this.sendRequest(GET, 'delegates/', data, callback);
-};
+LiskAPI.prototype.listActiveDelegates = privateApi.wrapSendRequest(GET, 'delegates', limit => ({ limit }));
 
 /**
  * @method listStandbyDelegates
  * @param limit
- * @param optionsOrCallback
- * @param callbackIfOptions
  *
  * @return API object
  */
 
-LiskAPI.prototype.listStandbyDelegates = function listStandbyDelegates(
-	limit, optionsOrCallback, callbackIfOptions,
-) {
-	const offset = typeof optionsOrCallback !== 'function' && optionsOrCallback.offset ? optionsOrCallback.offset : 101;
-	const orderBy = typeof optionsOrCallback !== 'function' && optionsOrCallback.orderBy ? optionsOrCallback.orderBy : 'rate:asc';
-	const callback = callbackIfOptions || optionsOrCallback;
-	const data = this.constructRequestData({ limit, offset, orderBy }, optionsOrCallback);
-	return this.sendRequest(GET, 'delegates/', data, callback);
-};
+LiskAPI.prototype.listStandbyDelegates = privateApi.wrapSendRequest(GET, 'delegates', limit => ({ limit }));
```

Comment:
This needs to incorporate `offset` and `orderBy` (it's the reason `getDataFn` takes two arguments).